standards/bug-reporting.md

```diff
@@ -31,9 +32,9 @@ A concise description of what the problem is. Pure description, no narrative or
 
 <b><u>Steps to Reproduce</u></b>
 1. Step by step instructions on how to reproduce this bug.
-2. Do not assume anything, 
-3. the more detailed your list of instructions, 
-4. the easier it is for the developer to track down the problem!
+2. Do not assume anything. 
+3. The more detailed your list of instructions. 
+4. The easier it is for the developer to track down the problem!
```

Comment:
I think 3 should keep the comma otherwise it reads kind of funky.

Reply:
True, i think i got stuck on grammar side  for a numbered line ending in comma etc...